gasometer/src/lib.rs

```diff
@@ -575,7 +589,8 @@ impl<'config> Inner<'config> {
 		}
 	}
 
-	fn gas_cost(
+	/// Returns the gas cost numerical value.
+	pub fn gas_cost(
```

Comment:
This doesn't seem to need to be public.

```suggestion
	fn gas_cost(
```